packages/community-cli-plugin/src/commands/bundle/bundle.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall react_native
+ */
+
+import type {Config} from '@react-native-community/cli-types';
+import type {CommandLineArgs} from './bundleCommandLineArgs';
+
+import buildBundle from './buildBundle';
+import bundleCommandLineArgs from './bundleCommandLineArgs';
+
+/**
+ * Builds the bundle starting to look for dependencies at the given entry path.
+ */
+export function bundleWithOutput(
+  _: Array<string>,
+  config: Config,
+  args: CommandLineArgs,
+  output: any, // untyped metro/src/shared/output/bundle or metro/src/shared/output/RamBundle
```

Comment:
nit: is it still untyped?

Reply:
Will be typed in an upcoming PR, that converts all files to `@flow strict-local` 👍🏻